plugins/populator/frontend/public/javascripts/countly.models.js

```diff
@@ -495,8 +627,8 @@
 
 	function updateUI(stats){
 		for(var i in stats){
-            totalStats[i] += stats[i];
-			$(".populate-stats-"+i).text(totalStats[i]);
+			totalStats[i] += stats[i];
+			$("#populate-stats-"+i).text(totalStats[i]);
```

Comment:
The rest of the code seems to be you reverting changes made by others including whole countly.views.js, localization and other files :D